internal/build/build.go

```diff
@@ -10,27 +10,33 @@ import (
 	"github.com/google/go-containerregistry/pkg/name"
 	"github.com/pkg/errors"
 
+	"github.com/buildpack/pack/internal/api"
 	"github.com/buildpack/pack/internal/builder"
 	"github.com/buildpack/pack/internal/cache"
 	"github.com/buildpack/pack/internal/style"
 	"github.com/buildpack/pack/logging"
 )
 
-// PlatformAPIVersion is the current Platform API Version supported by this version of pack.
-const PlatformAPIVersion = "0.1"
+const (
+	// MinPlatformAPIVersion is the minimum Platform API Version supported by this version of pack.
+	MinPlatformAPIVersion = "0.1"
```

Comment:
`Min` here seems to misconstrued how platform versions are supported. At least for `pre-stable`, each version is independent (not semver). [Here's a little more detail](https://github.com/buildpack/spec/blob/master/buildpack.md#buildpacktoml-toml) although only mentioned for Buildpack API it also should apply to Platform API version.

A better representation could be:
```
var SupportedPlatformAPIVersions := ["0.1", "0.2"]
```

Reply:
Good point, updated.